### DISCUSSION TITLE
add support for youtube handles in channel urls

### DISCUSSION THREAD

**Discussion by mattsblack** [ThioJoe/YT-Spammer-Purge#976] [Ideas]
I'm using the latest version (2.16.10) of the spammer purge program.

When I try to paste in a url with the new youtube handles it produces an error. I think that there should be a feature added to account for these new handles. It still works fine if I change the url to the old channel url format.

Example:

![Screenshot 2022-11-18 020324](https://user-images.githubusercontent.com/47698245/202641533-75377fce-5dfa-4aa1-94b1-6c81ccaf8823.png)

**Comment by Lightning11wins** [comment]
FYI: This is a [duplicate issue](https://github.com/ThioJoe/YT-Spammer-Purge/issues/974).

**Comment by ThioJoe** [comment]
It should work now with the latest Beta6

**Comment by Lightning11wins** [comment]
This new "helpful tools" feature is great. However, it also doesn't work on channel URLs with handles. Is it intended not to work on those kinds of URLs or should I make a bug report for that?

![image](https://user-images.githubusercontent.com/42805582/206943845-03b9de10-fa76-4c5f-b5c6-9e10bb2cff8e.png)

**Reply by ThioJoe** [reply]
That one only works with video URLs, so just pick a random video from the channel and enter that.

Also it might not be clear but you have to enter "1" as the choice to select that tool, that's why it says invalid choice. 

There's only one tool in the menu so it might be confusing.

It should work like this:

<img width="649" alt="image" src="https://user-images.githubusercontent.com/12518330/207081254-230da37c-5470-4531-99a0-cae5d2ed52c1.png">

**Reply by Lightning11wins** [reply]
Oh... Wow. Sorry. I guess I'm more tired than I thought!
